### ISSUE TITLE
Node debugging: Breakpoints are verified

### ISSUE THREAD

**Issue by tilman22** [Microsoft/vscode#67644]
My main folder is named "xyz.c++11.x32.root" (Linux). When I launch the node-debugger and try to set breakpoints, the breakpoints are listed as "Unverified breakpoint". 

When I rename the main folder to "xyz.root", then I can set breakpoints and the debugger stops at these breakpoints as expected.

**Comment by tilman22** [comment]
[The problem here seems to be the name of the folder/path - there does not seem to be a duplicate]

**Comment by weinand** [comment]
@roblourens are you setting breakpoints via a regular pattern?

**Comment by roblourens** [comment]
Yes, but I escape `+` and this works for me. If this is still an issue, can you set `"trace": true` in your launch config, run it again, and share the log from the path in the output channel?
